src/test/java/org/mitre/synthea/export/FHIRSTU3ExporterTest.java

```diff
@@ -64,6 +65,7 @@ public static void setup() {
     // Ensure Physiology state is enabled
     physStateEnabled = State.ENABLE_PHYSIOLOGY_STATE;
     State.ENABLE_PHYSIOLOGY_STATE = true;
+    PayerManager.loadPayers(new Location("Massachusetts", null));
```

Comment:
Ideally, you do not want to specify a location in the unit tests like "Massachusetts" because it will break synthea-international support. Instead, you want to do something like

```
String testStateDefault = Config.get("test_state.default", "Massachusetts");
PayerManager.loadPayers(new Location(testStateDefault, null));
```